ui/details-panel/SimilarJobsTab.jsx

```diff
@@ -0,0 +1,302 @@
+import PropTypes from 'prop-types';
```

Comment:
Could you add a `import React from 'react';` at the top of this file, so it doesn't cause linter errors when #3481 is merged?

Reply:
good call, done.